app/common/elasticsearchclient.py

```diff
@@ -243,17 +243,27 @@ def best_hit_search(self,
         #there are len(params.q) responses - one per query
         for i in range(len(results['responses'])):
             name = params.q[i]  #even though we are guaranteed that responses come back in order, and can match query to the result - this might be convenient to have       
+            print "name =" + name 
             lower_name = name.lower()
             res = results['responses'][i]
             exact_match = False
             if(res['hits']['total']>0):
                 hit = res['hits']['hits'][0] #expect either 1 result or none
                 highlight = ''
+                fields = kwargs['fields'];
```

Comment:
Yes semicolon is a habit hard to get rid of! I did not even notice I had one there